Clarifying the Cloud VPC add-on refers to infrastructure, not to a dedicated AWS account (#5641)
mattermost/docs: source/about/cloud-subscriptions.rst

Is Mattermost Cloud Enterprise a dedicated instance run on AWS systems?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Mattermost Cloud Enterprise can be deployed as a dedicated Mattermost environment running with separate infrastructure for that customer specifically, i.e. separate database, separate VMs, separate Kubernetes cluster. Please contact Mattermost Sales for more information on this option.

Reviewer: ```suggestion
Mattermost Cloud Enterprise can be deployed as a dedicated Mattermost environment running with separate infrastructure for your requirements (e.g., separate database, separate VMs, separate Kubernetes cluster). Please contact Mattermost Sales for more information on this option.
```
